mobidam-sst-management-frontend/frontend/src/api/ZuordnungService.ts

```diff
@@ -0,0 +1,68 @@
+import Zuordnung from "@/types/Zuordnung";
+import FetchUtils from "@/api/FetchUtils";
+import { Levels } from "@/api/error";
+import { useSnackbarStore } from "@/stores/snackbar";
+
+export default class ZuordnungService {
+    private static base: string | undefined = import.meta.env
+        .VITE_VUE_APP_API_URL;
+    public static create(instance: Zuordnung): Promise<Zuordnung> {
+        return fetch(
+            `${this.base}/api/zuordnung`,
+            FetchUtils.getPOSTConfig(instance)
+        )
+            .then((response) => {
+                useSnackbarStore().showMessage({
+                    message: "Speichern erfolgreich.",
+                    level: Levels.SUCCESS,
+                });
+                FetchUtils.defaultResponseHandler(response);
+                return response.json();
+            })
+            .catch((err) => {
+                useSnackbarStore().showMessage({
+                    message: "Speichern der Person fehlgeschlagen.",
+                    level: Levels.ERROR,
+                });
+                FetchUtils.defaultResponseHandler(err);
+            });
+    }
+
+    public static getZuordnungenByID(
+        instanceId: string | undefined
+    ): Promise<Zuordnung[]> {
+        return fetch(
+            `${this.base}/api/zuordnung/bySchnittstelle/${instanceId}`,
+            FetchUtils.getGETConfig()
+        ).then((response) => {
+            if (response.status != 502) {
```

Comment:
```suggestion
            if (response.status == 502) {
```
Warum wir hier auf ein ungleich geprüft? Müsste man nicht prüfen ob das 502 `Bad Gateway` zurückgekommen ist?